### ISSUE TITLE
[HELP] Does not run on Windows 9

### ISSUE THREAD

**Issue by andre-filho** [kelseyhightower/nocode#3893]

**Comment by MikeSim1** [comment]
[ ](url)

**Comment by Saiini** [comment]
`


`

In the 2090 update, stuff has changed where some libraries were deemed "malicious" from windows defender. Adding this to the env-variables should fix the issue.

**Comment by aditya-shrivastava2018** [comment]
Are you sure? Seems to support my Windows 83 & Windows 12 properly. Hold on, my pig flew off again, i'll be back to comment on the possible fixes you can do.

**Comment by andre-filho** [comment]
> Are you sure? Seems to support my Windows 83 & Windows 12 properly. Hold on, my pig flew off again, I'll be back to comment on the possible fixes you can do.

Yes, I am sure. It does not work no matter what I do. BTW, I sugest that you keep that RedBull out of reach from your pig, it won't do it any good.

Any progress on possible fixes?

**Comment by aditya-shrivastava2018** [comment]
> > Are you sure? Seems to support my Windows 83 & Windows 12 properly. Hold on, my pig flew off again, I'll be back to comment on the possible fixes you can do.
> 
> Yes, I am sure. It does not work no matter what I do. BTW, I sugest that you keep that RedBull out of reach from your pig, it won't do it any good.
> 
> Any progress on possible fixes?

So it seems the pig was smoking pot. Again. Crazy mofo.
Anyhoo-
What really fixed it for me was nuking the computer. I'll explain why it works - if you want noCode to run, then the computer should be code free as well. Makes sense comrade?
